.github/workflows/build_prodrc_pr.yaml

```diff
@@ -9,20 +9,25 @@ name: Build Prod RC Image
     - opened
     - synchronize
     - ready_for_review
+
 jobs:
-  docker_build:
-    runs-on: ubuntu-latest
-    steps:
-    - name: Check out GitHub Repo
-      if: github.event.pull_request.draft == false
-      with:
-        ref: "${{ github.event.pull_request.head.sha }}"
-      uses: actions/checkout@v2
-    - name: Build and Push to Packages
-      if: github.event.pull_request.draft == false
-      env:
-        PR: "${{ github.event.pull_request.number }}"
-        SHA: "${{ github.event.pull_request.head.sha }}"
-        DOCKER_ACTOR: "${{ secrets.GHCR_USERNAME }}"
-        DOCKER_TOKEN: "${{ secrets.GHCR_TOKEN }}"
-      run: "./.github/workflows/scripts/build_prodrc_pr.sh\n"
+  run_build_and_push:
```

Comment:
Each of the prod, dev, and truss workflows runs these three jobs -- build/push the narrative image, and then run the unit and integration tests against the docker image built in the previous step. It would be possible to eliminate the individual files for the prod / dev / truss workflows and just have a single file. The only advantage of separate files is that the name of the workflow (Build Prod RC Image, Build Dev Image, etc.) gets reported on the 'Actions' tab.